Source/WebKit/GPUProcess/media/RemoteMediaPlayerProxy.h

```diff
@@ -473,6 +470,9 @@ class RemoteMediaPlayerProxy final
 #if !RELEASE_LOG_DISABLED
     const Ref<const Logger> m_logger;
 #endif
+#if PLATFORM(COCOA)
+    LayerHostingContextManager layerHostingContextManager;
```

Comment:
Needs a `m_`. Can this be a `UniqueRef<>` so that we don't need to pull in `LayerHostingContextManager.h` into this header?